schema.sql

```diff
@@ -5,5 +5,6 @@ CREATE TABLE animals (
   escape_attempts INT,
   neutered BOOLEAN,
   weight_kg FLOAT,
+  species VARCHAR(20),
```

Comment:
- [ ] <b>REQUIRED:</b> The project requires you to add a new column, but not just add it in the same previous `CREATE TABLE` command, but instead with an `ALTER TABLE` command. Kindly do that to comply with project requirements.

Reply:
Ok thank you for the suggestion, I'll modify the file. 🤓